fundraiser/project/fundraiser-contract/src/data_structures/asset_info.sw

```diff
@@ -0,0 +1,13 @@
+library asset_info;
+
+/// Used to track the total amount pledged to an asset
+pub struct AssetInfo {
+    /// The amount that is currently pledged
+    amount: u64,
```

Comment:
If a struct only has one field, is the struct abstraction still useful? Could it just be a `u64` ?

Reply:
It could be a `u64` though standardizing into a structure also has some meaning. I do not recall right now if additional information is meant to be included in the struct in future updates since I have narrowed the scope to these improvements rather than feature updates.